docs/js-core-1/week-4/lesson.md

```diff
@@ -163,10 +163,10 @@ Did you notice how calling the concat method did not change `arr`? This is becau
 If you want to use the array returned by calling `.concat()` you should store it in a new variable.
 
 ```js
-const arr = [1, 2, 3];
-const newArr = arr.concat(4);
+const array = [1, 2, 3];
```

Comment:
Thanks for this!

I think this still violates the style guide https://syllabus.codeyourfuture.io/guides/code-style-guide#avoid-short-names

Here's the bit I think talks about this:

> Here are some examples of bad names that you should avoid:
> 
> Single letters like x or y
> Abbreviations like evt instead of event
> Generic names like array or string


What are your thoughts?

Reply:
okay i will change the array to temp[variable name]  is it okay?
newArray to newTemp